aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -318,9 +328,16 @@ def pg_lang(self) -> List[Dict[str, Any]]:
 
     @property
     def pg_roles(self) -> Dict[str, PGRole]:
+        filtered_roles = ["rdstopmgr"]
+        if self.filtered_roles:
+            filtered_roles.extend(self.filtered_roles)
+        roles_list = ",".join(f"'{role}'" for role in filtered_roles)
+
         if not self._pg_roles:
             # exclude system roles
-            roles = self.c("SELECT quote_ident(rolname) as safe_rolname, * FROM pg_catalog.pg_roles WHERE oid > 16384")
+            roles = self.c(
+                f"SELECT quote_ident(rolname) as safe_rolname, * FROM pg_catalog.pg_roles WHERE oid > 16384 AND rolname NOT IN ({roles_list})"
```

Comment:
This is a SQL injection waiting to happen - is there any reason for not using args? Quoting problems?

. Remember the aiven-db-migrate can be used as a library as well, so you never know the source of your statements. It's not just what you use from your CLI.

Reply:
Thank you for catching this! removed this potential SQL injection completely 😅